### PR TITLE
Plotting Feature

### PR DESCRIPTION
**Pull request by jwnorm**
### Plotting Feature

Added a new file _plotting.py_, which contains the function `export_hill_plot`. This is integrated in the main function of _hillmaker_, `make_hills`. This function is based on a code sample from @misken with some notable extensions:

- Explicitly declaring a plot style (seaborn-darkgrid)
- Infer number of days plotted based on bin_size_minutes
- Generalized plot labels and titles
- Added capacity line for occupancy plots (optional)
- Exports plots to png files in specified output directory

There are a few open items I would like to have resolved at some point:

- [ ] Consider generating plots by different categories if `catfield` is supplied
- [ ] Allow user to supply percentile to be plotted with the mean
- [ ] Create meaningful combinations of plots, i.e. combining arrivals and occupancy (thinking using facets instead of dual-axes)

### CODE REVIEW

File: src/hillmaker/hills.py [hunk 1]
```diff
@@ -1,6 +1,18 @@
 """Hillmaker"""
 
-# Copyright 2022 Mark Isken
+# Copyright 2015, 2022 Mark Isken, Jacob Norman
```
Comment (misken) [inline src/hillmaker/hills.py:3]: Take out the Apache stuff as I've moved hillmaker to an MIT License.

**Review by misken — CHANGES_REQUESTED** [review]
Take out the Apache related stuff as hillmaker is now MIT

File: src/hillmaker/hills.py (hunk 1 above)
Comment (jwnorm) [reply src/hillmaker/hills.py]: Fixed in most recent commit

**Review by misken — APPROVED** [review]
Thanks

**Comment by misken** [comment]
After I play around with the plotting functions, I'll do the changes to add the relevant args to the CLI.

**Pull request merged**